azure-pipelines/create-release.yml

```diff
@@ -0,0 +1,33 @@
+name: "GitHub & Web Release [$(SourceBranchName)] - $(Date:yyyyMMdd)$(Rev:.r)"
```

Comment:
Was the old process not working?  Looks like a full rewrite on the pipelines?

Reply:
a lot of the old process was manual.  A few of AzDO task has been updated to support the functionality we need so I removed the manual pieces.